iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/DanglingLineScalable.java

```diff
@@ -109,9 +109,9 @@ public void filterInjections(Network n, List<Injection> injections, List<String>
      * If scalingConvention is GENERATOR, the load active power decreases for positive "asked" and increases inversely
      */
     @Override
-    public double scale(Network n, double asked, Scalable.ScalingConvention scalingConvention) {
+    public double scale(Network n, double asked, ScalingContext context) {
```

Comment:
If we just add read only data in `ScalingContext`, it is not really a context. It is more a `ScalingParameter`